### PR TITLE
Added 'View latest version' button to the documentation section.

### PR DESCRIPTION
**Pull request by eXpl0it3r**
As by a number of requests, here comes the "View latest version!" button for the documentation. :smile: 

I google translated the French "Voir dernière version!", let me know what the proper naming should be.

### CODE REVIEW

**Comment by LaurentGomila** [comment]
I'd rather put a paragraph below the main title. Cleaner than embedding a button/link into a <h1> tag, and we have room to be more expressive -- something like "warning: this is the documentation of an old version of SFML. Click here to switch to the latest version".

**Comment by eXpl0it3r** [comment]
Like this?

**Comment by LaurentGomila** [comment]
Yes, looks good :+1:

**Comment by eXpl0it3r** [comment]
French translation? :smile:

**Comment by mantognini** [comment]
Attention: cette page se réfère à une ancienne version de SFML. Cliquez ici pour passer à la dernière version.

**Comment by eXpl0it3r** [comment]
Pushed the changes. The French site looks like this:

File: documentation/header.php [hunk 1]
```diff
@@ -8,6 +10,12 @@
...
     require('../../header.php');
+    
+    if($version != $latest)
+    {
+        $linklatest = '<div class="important"><strong>Warning:</strong> this page refers to an old version of SFML. Click <a href="../' . $latest . '/' . str_replace('documentation/' . $version . '/', '', $breadcrumbs[$pagetitle]) . '">here</a> to switch to the latest version.</div>';
+    }
 ?>
 
-<h1>Documentation of SFML <?php echo $version ?></h1>
+<h1>Documentation of SFML <?php echo $version; ?></h1>
```
Comment (mantognini) [inline documentation/header.php]: I've no idea if it's relevant or not, but on the same line in the french header there's no semicolon...

**Comment by mantognini** [comment]
Looks good!

I'm wondering, do we have some sort of checklist for releases? Because we now have to update this `$latest` setting as well.

**Comment by eXpl0it3r** [comment]
I once started one, but then I couldn't settle on a format and dropped it again. :wink:

File: documentation/header.php (hunk 1 above)
Comment (eXpl0it3r) [inline documentation/header.php]: It doesn't really matter, but I've also added it for the French page.

**Comment by mantognini** [comment]
Maybe a simple list in this repo Readme will do? For now, we could simply start it with this PR with only one item (the `$latest` var) and a warning that the list is not completed yet. Then, when updating the site for the next release we can complete it on the fly. What do you think?

**Comment by LaurentGomila** [comment]
$latest could be filled automatically with some PHP magic (taking the greatest number in documentation/ sub-folders).

**Comment by Foaly** [comment]
:+1: This is a really cool feature! It bugs me that the google search result always points to an outdated version. I usability suggestion: Maybe the whole text should be the link. It makes it easier to click, than the small "here".

**Comment by Bromeon** [comment]
...or the entire box, so it's more like a button.

**Comment by eXpl0it3r** [comment]
The whole box is now clickable.

**Comment by mantognini** [comment]
Looks good!

**Comment by Bromeon** [comment]
Thanks for adapting!

Just a minor aesthetic UX detail: it's slightly confusing that the text behaves like a link, although the whole box is a button. An alternative might have been to highlight the red background when hovered (instead of underlining the text).

**Pull request merged**